data_model/src/block.rs

```diff
@@ -92,13 +91,29 @@ mod model {
         /// Block header
         pub header: BlockHeader,
         /// Topology of the network at the time of block commit.
-        pub commit_topology: UniqueVec<peer::PeerId>,
+        pub commit_topology: Vec<peer::PeerId>,
         /// array of transactions, which successfully passed validation and consensus step.
         pub transactions: Vec<TransactionValue>,
         /// Event recommendations.
         pub event_recommendations: Vec<EventBox>,
     }
 
+    /// Signature of a block
+    #[derive(
+        Debug,
+        Clone,
+        PartialEq,
+        Eq,
+        PartialOrd,
+        Ord,
+        Decode,
+        Encode,
+        Deserialize,
+        Serialize,
+        IntoSchema,
+    )]
+    pub struct BlockSignature(pub u64, pub SignatureOf<BlockPayload>);
```

Comment:
I think you should write doc comment here to specify what `u64` means here. 
You might as well consider not using tuple struct here.